lib/saml/url_service.rb

```diff
@@ -1,74 +1,109 @@
 # frozen_string_literal: true
 
 module SAML
-  # This module is responsible for providing the URLs for the various SSO and SLO endpoints
-  module URLService
-    # converts from symbols to strings
-    SUCCESS_RELAY_KEYS = Settings.saml.relays&.keys&.map { |k| k.to_s }
-
-    # SSO URLS
-    def mhv_url(relay_state)
-      build_sso_url(authn_context: 'myhealthevet', relay_state: relay_state)
+  # This class is responsible for providing the URLs for the various SSO and SLO endpoints
+  class URLService
+    VIRTUAL_HOST_MAPPINGS = {
```

Comment:
I could see moving this to settings.yml because it's more about where the code is deployed than actual code. I'm thinking we could test this locally using https://ngrok.com/ or a similar service.

Reply:
what about adding a default?  we get a 500 if the url is not in the hash.  Not terribly likely, but could happen if a new virtual host is added.